src/plopp/graphics/canvas.py

```diff
@@ -115,14 +115,14 @@ def autoscale(self, draw=True):
     def draw(self):
         self.fig.canvas.draw_idle()
 
-    def savefig(self, filename: str, dpi: float = None):
+    def savefig(self, filename: str, **kwargs):
         """
         Save plot to file.
         Possible file extensions are `.jpg`, `.png` and `.pdf`.
         The default directory for writing the file is the same as the
         directory where the script or notebook is running.
         """
-        self.fig.savefig(filename, bbox_inches="tight", dpi=dpi)
+        self.fig.savefig(filename, **{**{'bbox_inches': 'tight'}, **kwargs})
```

Comment:
```suggestion
        settings = {'bbox_inches': 'tight'}
        settings.update(kwargs)
        self.fig.savefig(filename, **settings})
```
or something like that, so users can override our defaults?

Reply:
users can override the default with the syntax I used. If `bbox_inches` is in kwargs, it will take precedence over our default

Reply:
Oh, I didn't notice the nested '**{**'{...}'